MODULE.bazel

```diff
@@ -0,0 +1,71 @@
+module(
+    name = "io_bazel_rules_kotlin",
```

Comment:
Can you call this just "rules_kotlin"? This is what gets exported to BCR: https://github.com/bazelbuild/bazel-central-registry/tree/main/modules

So "io_bazel_" prefix would be a precedent.

"repo_name = " attribute controls how this can be used, that is what you need in the `load` statements and as dependencies.